app/assets/javascripts/miq_angular_application.js

```diff
@@ -5,6 +5,7 @@ ManageIQ.angular.app = angular.module('ManageIQ', [
 ]);
 miqHttpInject(ManageIQ.angular.app);
 
+ManageIQ.angular.rxSubject = new Rx.Subject();
```

Comment:
It depends on our needs, I'd go with `rxSubject` as event emitter, so multiple generic events can be sent trough this. So yes I'm planning to use this for non-toolbar things as well.